clang/lib/DPCT/DPCT.cpp

```diff
@@ -1090,6 +1273,33 @@ int runDPCT(int argc, const char **argv) {
     parseFormatStyle();
   }
 
+  if (MigrateCmakeScriptOnly) {
+    auto CmakeScriptLists = OptParser->getSourcePathList();
```

Comment:
Please consider put the functions, to simplify the DPCT.cpp main framwork.

Reply:
Okay